packages/cozy-search/README.md

```diff
@@ -0,0 +1,87 @@
+# Cozy-search
+
+### Prerequisite for both components
+
+1. Update `cozy-scripts` version of your app to 8.4.0.
+
+2. Add the provider in your tree :
+
+```jsx
+<DataProxyProvider>
+  { children }
+</DataProxyProvider>
+```
```

Comment:
Here I would mention that the SearchEngine is in the cozy-dataproxy-lib project, so it is easier to understand why the DataProxyProvider is needed.